packages/firebase_dynamic_links/android/build.gradle

```diff
@@ -45,7 +45,7 @@ android {
         disable 'InvalidPackage'
     }
     dependencies {
-        api 'com.google.firebase:firebase-dynamic-links:16.1.2'
+        api 'com.google.firebase:firebase-dynamic-links:16.1.7'
```

Comment:
Could we go to 16.1.8 here so that it is inline with [BoM 17.0.0](https://firebase.google.com/support/release-notes/android#firebase_bill_of_materials_firebase_bom_version_1700)

Reply:
ok